WPMedia/fusion-news-theme-blocks
# THEMES-1666/THEMES-1898: Accessible nav menu button and Greek language support

## Description

This PR adds in an accessible button that will close the header nav menu when it is open. Styling for the top of the nav menu has been updated to accommodate the button. Also the `Close` phrase has been moved to the global phrases block and it's uses updated. As a side effect of updating the translation files Greek language phrases have also been added.

#### Jira Ticket: [THEMES-1666](https://arcpublishing.atlassian.net/browse/THEMES-1666) [THEMES-1898](https://arcpublishing.atlassian.net/browse/THEMES-1898)

#### Related PR: https://github.com/WPMedia/arc-themes-manifests/pull/260

## Test Steps

1. Checkout this branch `git checkout THEMES-1666`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block,@wpmedia/global-phrases-block`
3. Navigate to a rendered page (like localhost/pf/homepage/?_website=the-gazette, trying this in the editor may take a while).
4. With a screen reader active (CMD + F5 on a Mac) navigate through the page using only the keyboard.
5. Open the section menu when you get to it, ensure the close button immediately gets focus.
6. Go through the menu items and make sure it loops back to the close button after reaching the end.
7. Activate the button with the keyboard. Make sure the nav closes and focus returns to the section menu button.
8. For Greek language support change the `"locale"` property in your `blocks.json` file to `"el"`.
9. Restart fusion with all blocks linked.
10. Ensure all translated phrases are showing in Greek.
 

[THEMES-1666]: https://arcpublishing.atlassian.net/browse/THEMES-1666?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[THEMES-1898]: https://arcpublishing.atlassian.net/browse/THEMES-1898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
